docs/js-core-1/week-4/homework.md

```diff
@@ -76,3 +76,9 @@ Have some extra time before our next class? Fill it with these harder challenges
 **Alert:** Some of these challenges might include concepts that we haven't taught you yet. You should us Google to try to learn how to solve them.
 
 https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week3
+
+## Writing a meeting request email (PD - 30 min)
+
+In the following scenario imagine you are a recruit for a company and your team leader asks you to write an email requesting a meeting with the Head of Customer Services (HCS). Remember to include the subject line and use appropriate formatting.
+
+"HCS has complained about the customer services application to the software team leader. He has found some frustrating problems with the user interface. The team leader told you to go and see the HCS, find out what is wrong and then come back with improvements. The HCS is very busy. You should email requesting a meeting and outline clearly what you needs the meeting for."
```

Comment:
"clearly what you need the meeting for" - no S on the need.

Reply:
Done